gatsby-node.js

```diff
@@ -110,25 +110,29 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   // get all the markdown files
   const result = await graphql(`
     {
-      posts: allFile(
+      posts: allMdx(
         filter: {
-          sourceInstanceName: { eq: "posts" }
-          ext: { in: [".md", ".mdx"] }
+          frontmatter: {
```

Comment:
can you keep this filtered based on folder. I am not sure what the right architecture is. eventually content should be in a seperate repo or a headless CMS.

We can do some research on that, and suggest a good architecture at a later point in time

Reply:
I don't think it is good for us to prefer headless cms here. If we choose headless cms, then again it requires our team members to have an account for adding/altering the content which I eventually don't want to happen.

I think, if we manage the content alone in another repo, we should be good to go. Serverless doing the same!

Reply:
it might be the best idea to have every person run our website to add a blog.
anyways, this is not important now. so letting this be for now

Reply:
Basically anyone who has github account and part of codebrahma team in github, should be able to update content on website by raising a PR.